packages/components/src/MaskedInput.tsx

```diff
@@ -34,7 +34,7 @@ type MaskedInputProps = {
   /** One or more examples of valid values. Used when deciding if next keystroke is valid (as rest of the current value may be incomplete) */
   example: string | string[];
   /** The current selection to use for the input */
-  selection: SelectionSegment | null;
+  selection?: SelectionSegment | null;
```

Comment:
If it's optional, there shouldn't be a need for `null` as well:

```suggestion
  selection?: SelectionSegment;
```

Reply:
Yep. I think at times when converting these I erred on adding `| null` since we use null to indicate that we explicitly set no value sometimes. But in this case it's just the starting value in the `TimeInput` component so I set it to undefined.